stylesheets/components/inputs/inputs.css

```diff
@@ -0,0 +1,87 @@
+@import "/stylesheets/index.css";
+
+.textbox {
+    background: var(--textbox-bg);
+    padding: 1rem;
+    resize: none;
+}
+
+.textbox.resize {
+    resize: both
+}
+
+.textbox-curved {
+    border-radius: 1rem;
+}
+
+.labelled-input {
+    display: flex;
+    flex-direction: column;
+}
+
+label {
+    display: block;
+}
+
```

Comment:
keep check on specificity , this can be tagged to all the labels in the whole html page

Reply:
Oh right. Thanks for pointing this out! Will change it.